modules/installation-custom-ibm-cloud-vpc.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-ibm-cloud-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-ibm-cloud-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into the subnets of an existing IBM Virtual Private Cloud (VPC). By deploying {product-title} into an existing VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
+
+Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf. You must configure networking for the subnets that you install your cluster to yourself.
+
+[id="installation-custom-ibm-cloud-vpc-requirements_{context}"]
+== Requirements for using your VPC
+
+The union of the VPC CIDR block and the machine network CIDR must be non-empty. The subnets must be within the machine network.
+
+You must correctly configure the existing VPC and its subnets before you install the cluster. The installation program does not create the following components:
+
+* NAT gateways
+* Subnets
+* Route tables
+* VPC network
+
+The installation program cannot:
+
+* Subdivide network ranges for the cluster to use.
+* Set route tables for the subnets.
+* Set VPC options like DHCP.
+
+include::snippets/custom-dns-server.adoc[]
+
+[id="installation-custom-ibm-cloud-vpc-validation_{context}"]
+== VPC validation
+
+To ensure that the subnets that you provide are suitable, the installation program confirms the following data:
+
+* All the subnets that you specify exist.
+* You provide one subnet for control plane machines and one subnet for compute machines.
+* The subnet's CIDRs belong to the machine CIDR that you specified.
+
+[id="installation-custom-ibm-cloud-vpc-permissions_{context}"]
```

Comment:
The 'Division of permissions' section can be removed. I don't see any value in what is being communicated. We are investigating supporting the specification of a unique resource group for the existing vpc and subnets and have cluster resources provisioned in unique resource groups. Azure has this support. Perhaps when we have that support we can add this section back with suggestions on how to leverage that support.

Reply:
Removed.